.github/workflows/ci_bench_block_execution.yaml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   build-binaries:
+    if: contains(github.event.pull_request.labels.*.name, 'performance')
```

Comment:
If this expects to match with the PR title this should be `perf` (see [this](https://github.com/lambdaclass/ethrex/blob/main/.github/workflows/lint_pr_title.yml#L28)). If I'm mistaken ignore the comment.